templates/try-matrix.html

```diff
@@ -12,10 +12,13 @@ <h1>{{ page.title }}</h1>
         <div class="step choose-a-client">
             <div class="instructions">
                 <h3>Choose a client</h3>
-                <p>We recommend Element as a safe default, but there is a variety of
-                    clients available.
+                <p>
+                    We recommend Element as a safe default, but there is a variety of clients available.
                 </p>
-                <a href="/ecosystem/clients" class="call-to-action">Browse clients</a>
+                <div class="cta-wrapper">
+                    <a href="/ecosystem/clients/element/" class="call-to-action">Install Element</a>
```

Comment:
May I suggest a different wording here? "Install Element" sounds like a high hurdle already.  And at least for Element-web, that is not needed (even if you decide to install something later). "Use Element" might be a compromise that sounds like the hurdle is much lower (just open it in a webbrowser, hey!).

In the case of mobile, users would likely want to install an app though...

Reply:
Only playing a little stupid, but "[Browse clients]" might also be suboptimal. **A** `browse client ` is e.g. hydrogen or element-web :-). Something like "Discover more clients"?